packages/story-editor/src/components/canvas/emptyStateLayer.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import {
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Button,
+  Icons,
+} from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '@web-stories-wp/i18n';
+import { useRightClickMenu } from '../../app';
+import useStory from '../../app/story/useStory';
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../elements/utils/createPage';
+import { Layer, PageArea } from './layout';
+import { shallowEqual } from '@web-stories-wp/react';
+
+const DisplayPageArea = styled(PageArea)`
+  position: absolute;
+  z-index: 1;
+`;
+
+const EmptyStateMessage = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const isEmptyStory = (pages) => {
+  if (pages.length === 0) {
+    return false;
+  }
+
+  const hasOnlyOnePage = pages.length === 1;
+  const hasNoExtraElements = pages[0].elements.length === 1;
+  const hasNoBackgroundMedia = !('resource' in pages[0].elements[0]);
+  const hasDefaultBackgroundColor = shallowEqual(
+    DEFAULT_PAGE_BACKGROUND_COLOR.color,
+    pages[0].backgroundColor.color
+  );
+
+  return (
+    hasOnlyOnePage &&
+    hasNoExtraElements &&
+    hasNoBackgroundMedia &&
+    hasDefaultBackgroundColor
+  );
+};
+
+function EmptyStateLayer() {
+  const onOpenMenu = useRightClickMenu((value) => value.onOpenMenu);
```

Comment:
Can we hack this to open the menu in a fixed position rather than at the mouse cursor? It looks better with the menu centered on the page rather than shooting off from wherever you click on the button (that's not normally how popup menus work).

Again, compare Figma vs editor. The Figma menu actually isn't centered either, but not really sure why that is. It does look better though because the entire menu is inside the page area.

| Figma | Editor |
|-|-|
| ![Screen Shot 2022-01-19 at 17 56 49](https://user-images.githubusercontent.com/637548/150232118-3dbdbad2-4f68-4215-b967-b5694762d0b8.png) | ![Screen Shot 2022-01-19 at 17 57 01](https://user-images.githubusercontent.com/637548/150232127-49e15f49-96f0-41b0-8fe3-99ed4e8835fa.png) |

Reply:
Something like this https://github.com/google/web-stories-wp/pull/10261/commits/3d5c7a68f8a12b81ff424deb5b0440d3710dc6d8 ?

Reply:
Looks perfect! 🎯
![Screen Shot 2022-01-21 at 17 04 04](https://user-images.githubusercontent.com/637548/150606271-e369e710-ac5e-40e1-a00d-9c46c046801b.png)